torchgeo/datasets/utils.py

```diff
@@ -607,18 +611,22 @@ def percentile_normalization(
     return img_normalized
 
 
-def path_is_vsi(path: Path) -> bool:
-    """Checks if the given path is pointing to a Virtual File System.
+def path_is_gdal_vsi(path: Path) -> bool:
```

Comment:
I'm still considering deleting this function. It's only used in a single place, and it's only 1 line long, so we could just replace the if-statement on line 693 with this line. I would like to keep the documentation somehow though.

Reply:
Thats a lot of documentation to have in-line in the other method :P